python_modules/dagster/dagster/core/definitions/events.py

```diff
@@ -268,6 +268,7 @@ class AssetObservation(
         "_AssetObservation",
         [
             ("asset_key", AssetKey),
+            ("description", Optional[str]),
```

Comment:
maybe these core changes should be part of a separate diff than the frontend/graphql changes?

Reply:
I'll make description a `@property` for now and create a separate PR to add the description field. Label is a required field to render the log table.